nix/modules/writers.nix

```diff
@@ -76,12 +76,11 @@
         KEEP_VARS="''${KEEP_VARS:-}"
 
         unsetVars=$(
-          PATH=${pkgs.coreutils}/bin:${pkgs.findutils}/bin \
-            comm \
-              <(env | cut -d = -f 1 | sort) \
-              <(echo "${keepVars} $KEEP_VARS" | tr " " "\n" | sort) \
-              -2 \
-              -3
+          comm \
```

Comment:
> '/nix/store/dc66y6s8q1fl6y9mwvqbhw9h6dqqiv34-release-automation/bin/release-automation: line 21: comm: command not found

Reply:
fixed